src/highscores/highscores.js

```diff
@@ -0,0 +1,56 @@
+import globalStyles from "../styles.scss";
+import styles from "./highscores.scss";
+import {
+  setDataInStorage, 
+  getDataFromStorage
+} from '../utils/localStorage/localStorage';
+import { displayModal } from "../game/gameOverModal";
+
+export const saveMatchResult = (matchResult, points) => {
+  const playerName = getDataFromStorage('playerName') ?? 'Player';
+  const row = {name: playerName, score: points, status: matchResult};
+  const highScore = getDataFromStorage('highScore');
+    if (highScore) {
+      const highScores = [...JSON.parse(highScore), row].sort((a, b) => b.score - a.score).slice(0, 10);
+      setDataInStorage('highScore', JSON.stringify(highScores));
+    } else {
+      const newHighScores = [row];
+      setDataInStorage('highScore', JSON.stringify(newHighScores));
+    }
+}
+
+const displayHighScore = () => {
+  const highScores = JSON.parse(getDataFromStorage('highScore'));
+  const names = [...document.querySelectorAll('.table__data-name')];
```

Comment:
You are aware that `querySelectorAll` returns an array, right? So why destructure it?